pkg/hoist/hoist_launchable.go

```diff
@@ -74,7 +74,17 @@ func (hl *Launchable) If() launch.Launchable {
 	return LaunchAdapter{Launchable: hl}
 }
 
+func (hl *Launchable) IsOneoff() bool {
```

Comment:
cool, that was easy. Can you replace the other reference to the `IsUUIDPod` in this file in a separate commit? It's on line 322.